drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -544,15 +610,52 @@ bool LX200AstroPhysicsExperimental::ISNewSwitch(const char *dev, const char *nam
     if (!strcmp(name, UnparkFromSP.name))
     {
         IUUpdateSwitch(&UnparkFromSP, states, names, n);
-        int unparkPos = IUFindOnSwitchIndex(&UnparkFromSP);
-
-        LOGF_DEBUG("Unpark from pos set to (%d).", unparkPos);
+        ParkPosition unparkPos = (ParkPosition)IUFindOnSwitchIndex(&UnparkFromSP);
```

Comment:
use static_cast<...>

Reply:
I changed all occurrences to  static_cast(<ParkPosition>...)